tests/manager/manager-ctor-write_concern-error-001.phpt

```diff
@@ -12,12 +12,12 @@ echo throws(function() {
     create_test_manager(null, ['w' => 1.0]);
 }, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
 
-/* Note: Values of w < -1 are invalid, but libmongoc's URI string parsing only
+/* Note: Values of w < 0 are invalid, but libmongoc's URI string parsing only
  * logs a warning instead of raising an error (see: CDRIVER-2234), so we cannot
  * test for this. */
```

Comment:
Comment appears incorrect. libmongoc [returns an error](https://github.com/kevinAlbs/mongo-c-driver/blob/56a89c5359c165420cdca5df4652737cb8c80886/src/libmongoc/src/mongoc/mongoc-uri.c#L1733) for unsupported values, e.g. `w=-1`:

```php
echo throws(function() {
    create_test_manager("mongodb://localhost:27017/?w=-1", []);
}, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
```
Results in output:
```
Failed to parse MongoDB URI: 'mongodb://localhost:27017/?w=-1'. Error while parsing the 'w' URI option: Unsupported w value [w=-1].
OK: Got MongoDB\Driver\Exception\InvalidArgumentException
```

Reply:
Thank you for pointing this out - I guess that comment was never updated after CDRIVER-2234 was fixed.